server/src/main/java/model/placement/board/Board.java

```diff
@@ -118,7 +119,7 @@ private void surroundWithBuffer(Coordinates mastCoordinates) {
         HashSet<Coordinates> neighbours = new HashSet<>();
         for (int toAddToX = -1; toAddToX <= 1; toAddToX++) {
```

Comment:
Omg how much level of indentations ?

Reply:
Iterating over 2D matrix is inherently n^2 and nested for loop is a common idom for operating on 2D. There's no point in modifying this method.

Reply:
this method is has an exact duplicate in ShootingBoard class

Reply:
We're gonna address this problem as it's an obvious DRY violation, but I'm not the one to call the shots so it has to wait for @totine